### PR TITLE
Add ember-lts-3.20 to ember-try test matrix 

### PR DESCRIPTION
**Pull request by takshch**
Issue - #56 
Edited ember-try.js and ci.yml

### CODE REVIEW

**Comment by takshch** [comment]
@ijlee2 Please review #56

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -157,6 +157,7 @@ jobs:
     strategy:
       matrix:
         scenario:
+          - 'ember-lts-3.20'
```
Comment (ijlee2) [inline .github/workflows/ci.yml:160]: Can we list `ember-lts-3.20` after `ember-lts-3.16`? (personal preference)

**Review by ijlee2 — CHANGES_REQUESTED** [review]
This project uses `yarn` for package manager so please make sure `package-lock.json` isn't committed to this pull request.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (takshch) [reply .github/workflows/ci.yml]: Done!

**Comment by takshch** [comment]
@ijlee2 Sir please review it.

**Review by ijlee2 — APPROVED** [review]
Thank you for working on checking `ember-lts-3.20` in CI! 🎉

**Pull request merged**